content/docs/guides/continuous-delivery/github-actions.md

```diff
@@ -11,121 +11,140 @@ aliases:
-or promotions between different environments by merging or directly committing changes.
+Pulumi's [GitHub Actions](https://developer.github.com/actions) help you deploy apps and
+infrastructure to your cloud of choice, using nothing but code in your favorite language
+and GitHub. This includes previewing, validating, and collaborating on proposed
+deployments in the context of Pull Requests, and triggering deployments or promotions
+between different environments by merging or directly committing changes.
 
 Let's see how to get started -- it's easy!
 
 > **Note**: GitHub Actions is currently in public beta. Please [register for
-> access](https://github.com/features/actions) to start using the feature. Until you've been added, the
-> **Actions** tab won't show in your repos, and the `.pulumi/main.workflow` file described
-> below will not be recognized by GitHub. Also note that these features only work on
-> private repos in the specific account that has been granted access.
+> access](https://github.com/features/actions) to start using the feature. Until your
+> GitHub organization has been approved for the beta, the **Actions** tab won't show in
+> your repos, and the Actions-related files described below will not be recognized by
+> GitHub.
 
 # Pre-Requisites
 
-Before proceeding, you'll need to [Sign Up for Pulumi](https://app.pulumi.com/) (if you haven't already).
+Before proceeding, you'll need to [Sign Up for Pulumi](https://app.pulumi.com/) (if you
+haven't already). This guide also assumes you've reviewed the [GitHub Actions
+documentation](https://help.github.com/en/categories/automating-your-workflow-with-github-actions)
+and are generally familiar with its concepts and syntax.
 
-For your workflow to do anything interesting, you'll want to create a new Pulumi project.
-There are three ways to do this:
+For your workflow to do anything interesting, you'll want to create a new Pulumi project
+for it. There are three ways to do this:
 
 1. [Clone an existing Pulumi example](https://github.com/pulumi/examples)
 2. [Use the New Project wizard](https://app.pulumi.com/site/new-project)
-3. [Download the CLI]({{< relref "/docs/get-started/install" >}}) and run `pulumi new` to select a template
+3. [Download the CLI]({{< relref "/docs/get-started/install" >}}) and run `pulumi new` to
+   select a template
 
 # Creating a Workflow
 
-Although the full power of the Pulumi CLI is available to use with GitHub Actions, we recommend starting with
-our standard workflow, which consists of two actions, both triggered by GitHub events:
+Although the full power of the Pulumi CLI is available to use with GitHub Actions, we
+recommend starting with our standard workflow, which consists of two workflow files, each
+triggered by common GitHub events:
+
+1. **Pulumi Preview** runs `pulumi preview` in response to a Pull Request, showing what
+   would happen if the PR were merged into the target branch.
```

Comment:
Maybe change to "showing a preview of the changes to the target branch when the PR gets merged"?